roles/pbench_storage/templates/pbench-agent.cfg.j2

```diff
@@ -0,0 +1,52 @@
+[DEFAULT]
+pbench_install_dir = /opt/pbench-agent
+# CHANGE ME1!
+pbench_results_redirector = {{ pbench_results_redirector }}
+# CHANGE ME2!
+pbench_web_server = {{ pbench_web_server }}
+
+[pbench-agent]
+install-dir = %(pbench_install_dir)s
+pbench_user = pbench
+pbench_group = pbench
+pbench_run = /var/lib/pbench-agent
+pbench_log = %(pbench_run)s/pbench.log
+
+[results]
+user = pbench
+host_path = http://%(pbench_result_redirector)s/pbench-archive-host
+ssh_opts = -o StrictHostKeyChecking=no
+scp_opts = -o StrictHostKeyChecking=no
+webserver = %(pbench_web_server)s
+host_info_url = http://%(webserver)s/pbench-results-host-info.versioned/pbench-results-host-info.URL001
+dir = /pbench/public_html/incoming
+
+[pbench/tools]
```

Comment:
Is this the correct values for the intervals? @chaitanyaenr

Reply:
We set it to 10 while registering the tools.